### DISCUSSION TITLE
How to restore Windows 10

### DISCUSSION THREAD

**Discussion by stfulua** [Atlas-OS/Atlas#1225] [Help & Support] [answered]
Hello, i would like to restore my sistem to windows 10. The Atlas-OS has really been really cool, but it's time because it has many glitches. For almost an year, and i would like now to uninstall it but i don't know how to. Can somebody help?!

**Comment by Ast3risk-ops** [comment, marked as answer]
Uninstall guide: https://docs.atlasos.net/getting-started/reverting-atlas/
